environments/kubernetes/main.jsonnet

```diff
@@ -10,4 +10,5 @@ local app =
 { ['thanos-receive-controller-' + name]: app.thanos.receiveController[name] for name in std.objectFields(app.thanos.receiveController) } +
 { ['thanos-querier-cache-' + name]: app.thanos.querierCache[name] for name in std.objectFields(app.thanos.querierCache) } +
 { ['telemeter-' + name]: app.telemeterServer[name] for name in std.objectFields(app.telemeterServer) } +
+{ ['memcached-' + name]: app.memcached[name] for name in std.objectFields(app.memcached) } +
```

Comment:
It might be helpful to prefix this with `telemeter-memcached-` until we moved those manifests back to Telemeter or its own repo.